src/pie_modules/taskmodules/common/mixins.py

```diff
@@ -257,9 +257,13 @@ def get_statistics(self) -> Dict[Tuple[str, str], int]:
             return {}
 
     def format_statistics(self, statistics: Dict[Tuple[str, str], int]) -> str:
+        # Statistics are converted to Series and then unstacked giving us the right DataFrame we need.
```

Comment:
"the right DataFrame we need" - that doesn't say anything! what is the right DataFrame we need?
EDIT: after fixing, move into if case (see below)